pom.xml

```diff
@@ -77,19 +77,19 @@
 		<dependency>
 			<groupId>com.fasterxml.jackson.core</groupId>
 			<artifactId>jackson-annotations</artifactId>
-			<version>2.7.5</version>
+			<version>[2.9.7,)</version>
```

Comment:
Having completely open ended versions can introduce unexpected regressions.

It's probably best to only limit it to minor version updates like we've been doing with python.

[2.9.7,2.10)

https://docs.oracle.com/middleware/1212/core/MAVEN/maven_version.htm#MAVEN402

Though it both jackson and tika are generally safer dependencies for upgrades than other java dependencies.

Reply:
I see, that was the github suggestion. I can look at it but I think it's better to just fix the version. Some of these will get much newer versions than the suggestions.